src/winforms/toga_winforms/factory.py

```diff
@@ -17,9 +17,10 @@
 from .widgets.scrollcontainer import *
```

Comment:
At column 1: (F403) 'from .widgets.scrollcontainer import *' used; unable to detect undefined names